lib/passport-saml/saml.js

```diff
@@ -10,7 +10,11 @@ var xmlbuilder = require('xmlbuilder');
 var xmlenc = require('xml-encryption');
 var xpath = xmlCrypto.xpath;
 var InMemoryCacheProvider = require('./inmemory-cache-provider.js').CacheProvider;
-var Q = require('q');
+
+/* global Promise */
```

Comment:
We can get around this if we mark `"es6": false` in the `package.json`. Up to the maintainers :)

Reply:
I find the ESLint docs here confusing: https://eslint.org/docs/user-guide/configuring#top I'll say this: Our minimum Node.js version we need to support is now Node 10. I'm opening to changing any esLint directives that are aligned with that.